virtualenv.py

```diff
@@ -1452,6 +1452,7 @@ def install_activate(home_dir, bin_dir, prompt=None):
             'activate.bat': ACTIVATE_BAT,
             'deactivate.bat': DEACTIVATE_BAT,
             'activate.ps1': ACTIVATE_PS,
+            'activate.xsh': ACTIVATE_XSH,
```

Comment:
I think this needs to actually be defined, thought apparently all the others are defined [here](https://github.com/pypa/virtualenv/blob/ed553318227ed1ec6415b6785eac9add084c7c21/virtualenv.py#L1981), which is some crazy nonsense I don't understand.

Reply:
The `ACTIVATE_xxx` variables hold base64-encoded copies of the actual activate scripts. They are embedded in the `virtualenv.py` script so that we don't need to ship them as separate files (the `virtualenv_embedded` directory isn't shipped as part of the wheel).

`bin/rebuild-script.py` rebuilds `virtualenv.py` from the files in the `virtualenv_embedded` directory. Some fiddling will be needed to add `ACTIVATE_XSH` to that mechanism (but I don't know precisely what).

Reply:
@pganssle - you mentioned you had this working somewhere?

Reply:
@scopatz I made a PR against your branch [here](https://github.com/scopatz/virtualenv/pull/1)